application/web/web.js

```diff
@@ -44,6 +44,7 @@ function createJournalEntries(){
                 <img src="./assets/vdots.svg">
             </button>`;
         journalEntries.appendChild(entryElement);
+        document.getElementById("no-entry-text").style.display = "none";
```

Comment:
Make sure to add some comments/context about what this line of code is doing and how it functions.